extensions.h

```diff
@@ -59,9 +74,17 @@ void hibaext_free(struct hibaext *ext);
  * The sshbuf will be consumed. */
 int hibaext_decode(struct hibaext *ext, struct sshbuf *blob);
 
-/* Encode a HIBA extension into a serialized blob to be included in a
- * certificate. blob must be already allocated. */
-int hibaext_encode(const struct hibaext *ext, struct sshbuf *blob);
+/* Encode one or more HIBA extensions into a serialized blob to be included in a
+ * certificate (either raw or base64 encoded).
+ * blob must be already allocated, and will be reset.
+ * It is invalid to request encoding of more than 1 identity extension, since a
+ * host cannot expose multiple identities.
+ *
+ * The default hibaext_encode points to the base64 version, uncompressed single
+ * extension for backward compatibility. */
+#define hibaext_encode(ext, blob) hibaext_encode_b64((const struct hibaext**)&ext, 1, 0, blob)
```

Comment:
The reference-taking in the macro feels a bit dodgy. Why don't we expose `hibaext_encode_one_raw` as `hibaext_encode`?

Nit: the `ext` might need parentheses.

Reply:
Instead I made the  hibaext_encode a small wrapping function (avoids increasing the breadth of entry points).